tests/containers/bci_tests.pm

```diff
@@ -69,6 +69,11 @@ sub run {
     my ($self) = @_;
     $self->select_serial_terminal;
 
+    if (get_var('HOST_VERSION') !~ '15-SP3') {
+        record_info('poo#98183', 'Test not supported yet on this host. poo');
```

Comment:
what "poo" at the end suppose to mean ? :D

Reply:
leftover